python_modules/libraries/dagster-airflow/dagster_airflow/dagster_factory.py

```diff
@@ -38,16 +39,19 @@ def make_definition_from_dag_bag():
     Args:
         dag_bag (DagBag): Airflow DagBag Model
         connections (List[Connection]): List of Airflow Connections to be created in the Airflow DB
+        definitions_kwargs (Optional[dict]): kwargs to be passed to the Definitions constructor
 
     Returns:
         Definitions
     """
+    definitions_kwargs = check.opt_dict_param(definitions_kwargs, "definitions_kwargs")
     schedules, jobs = make_schedules_and_jobs_from_airflow_dag_bag(
         dag_bag,
         connections,
     )
 
     return Definitions(
+        *definitions_kwargs if definitions_kwargs else (),
```

Comment:
Does this even work?

I've only ever seen `Definitions(arg1=arg1, arg2=arg2, **kwargs)` work. 

This needs unit tests either way.